service/src/main/resources/application.yaml

```diff
@@ -0,0 +1,60 @@
+logingw:
+  rest:
+    # Rest General Config
+    jwt:
+      exp-time: 4
+      alg-name: "RS256"
+    config:
+      some-key: "BETA"
+
+    # Users (config-defined)
+    users:
+      known-users:
+        -
+          username: "user1"
+          password: "password1"
+          groups:
+        -
+          username: "TestUser"
+          password: "password123"
+          email: "test@abs.com"
+          groups:
+            - "groupA"
+            - "groupB"
+            -
```

Comment:
I think this is unintended - with empty `-` I am getting an error

Reply:
Thank you! Yes, it was unintended (newline in editor auto-created another `-`-prefixed field